package-lock.json

```diff
@@ -11412,6 +11412,12 @@
           "integrity": "sha512-GSmOT2EbHrINBf9SR7CDELwlJ8AENk3Qn7OikK4nFYAu3Ote2+JYNVvkpAEQm3/TLNEJFD/xZJjzyxg3KBWOzw==",
           "dev": true
         },
+        "prettier": {
```

Comment:
The lock file shouldn't have changed as no dependencies were added/updated/removed. Let's not update it unless there is a reason this is needed.